tools/build_defs/preload.bzl

```diff
@@ -0,0 +1,4 @@
+# These are symbols that are available in newer versions of Buck, but not the open source one.
+
+def oncall(_team):
```

Comment:
Q: Shouldn't we use `tools/build_defs/oss/build_defs.bzl` as we're doing for other similar functions?

Reply:
Agreed that oss makes more sense - have put it as oss/preload.bzl instead